### PR TITLE
DlgCorruption

### PR DESCRIPTION
**Pull request by stevenndungu**
@africanmathsinitiative/developers this is ready for review

### CODE REVIEW

**Comment by maxwellfundi** [comment]
@stevenndungu  well done for this. @dannyparsons  could you also indicate where these lie on frmMain so that he could add them there?

**Comment by dannyparsons** [comment]
Thanks guys. I can't remember if I am in the middle of editing frmMain right now. I will get back to you on the menu items. It can probably be done on a separate branch later.

**Comment by maxwellfundi** [comment]
Great. You are most welcome.

**Comment by stevenndungu** [comment]
Fixes #2735  #2734 #2738

File: instat/dlgDefineCorruptionOutputs.vb [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    Public Sub SetCurrentColumn(strColumn As String, strDataFrame As String)
+        strSelectedColumn = strColumn
+        strSelectedDataFrame = strDataFrame
+        bUseSelectedColumn = True
+    End Sub
+
+    Private Sub SetDefaultColumn()
+        ucrSelectorCorruptionOutputs.ucrAvailableDataFrames.cboAvailableDataFrames.SelectedItem = strSelectedDataFrame
+        ucrReceiverCorruptionOutputs.Add(strSelectedColumn, strSelectedDataFrame)
+        bUseSelectedColumn = False
```
Comment (lilyclements) [inline instat/dlgDefineCorruptionOutputs.vb]: Out of interest, what does this sub add? I've noticed it in all three of these and would like to know why we are having this on these corruption dialogs

Comment (stevenndungu) [inline instat/dlgDefineCorruptionOutputs.vb]: It is a sub that helps in auto-filling  the receiver of a dialogue that opens on right click. All the three dialogues are to be added on right clicks ....like that of Make Date dialogue on right click

**Pull request merged**